app/src/main/res/drawable/rounded_bg_focused_menu_item.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+    <stroke
+        android:width="2dp"
+        android:color="#5F006F" />
+    <corners android:radius="32dp" />
```

Comment:
Please extract all resource into color and dimen files

Reply:
done